test/ssl-tests/20-cert-select.conf.in

```diff
@@ -104,3 +119,91 @@ our @tests = (
         },
     }
 );
+
+
+my $server_tls_1_3 = {
+    "ECDSA.Certificate" => "\${ENV::TEST_CERTS_DIR}${dir_sep}server-ecdsa-cert.pem",
+    "ECDSA.PrivateKey" => "\${ENV::TEST_CERTS_DIR}${dir_sep}server-ecdsa-key.pem",
+    "MinProtocol" => "TLSv1.3",
+    "MaxProtocol" => "TLSv1.3"
```

Comment:
I'm wondering whether this should just be "MinProtocol", i.e. no "Maxprotocol". Wouldn't make any difference until TLSv1.4 though, so not a big deal.